project/Dependencies.scala

```diff
@@ -0,0 +1,27 @@
+import sbt._
+
+object Dependencies {
+  val borerVersion = "1.6.3"
+  val akkaVersion = "2.6.13"
+
+  val scalaTest = "org.scalatest" %% "scalatest" % "3.2.2"
+  val akkaTyped = "com.typesafe.akka" %% "akka-actor-typed" % akkaVersion
```

Comment:
Pls make it a Provided depenendcy while you're here (so that it's assumed that it's provided in the classpath by the library's clients, and not pulled in automatically by this very library itself; this helps avoid version conflicts)

Reply:
Same for akka-stream; test dependencies don't matter, only the production ones

Reply:
Also, rel: https://github.com/VirtusLab/akka-safer-serializer/issues/15